data-ingestion-service/src/main/java/gov/cdc/dataingestion/rawmessage/service/RawELRService.java

```diff
@@ -21,19 +21,33 @@ public class RawELRService {
     private static final String CREATED_BY = "admin";
     @Value("${kafka.raw.topic}")
     String topicName;
+
+    @Value("${kafka.raw.xml-topic}")
+    String rawXmlTopicName;
+
     private final IRawELRRepository rawELRRepository;
     private final KafkaProducerService kafkaProducerService;
 
 
     public String submission(RawERLDto rawERLDto, String version) {
         RawERLModel created = rawELRRepository.save(convert(rawERLDto));
-        kafkaProducerService.sendMessageFromController(
-                created.getId(),
-                topicName,
-                rawERLDto.getType(),
-                0,
-                rawERLDto.getValidationActive(),
-                version);
+        if(created.getType().equalsIgnoreCase("HL7")) {
+            kafkaProducerService.sendMessageFromController(
+                    created.getId(),
+                    topicName,
+                    rawERLDto.getType(),
+                    0,
+                    rawERLDto.getValidationActive(),
+                    version);
```

Comment:
Can you handle this Version and pass it to lower stream as well.
We will be using this version variable to trigger the RTI flow
Logic
 - If version is 2 then flow will be directed to RTI
 - if version is empty or 1 it will be directed to Legacy Batch Flow